src/com/company/Main.java

```diff
@@ -8,18 +8,40 @@
 
 
 public class Main {
-    public static void main(String[] args){
+    public static void main(String[] args) {
+
+        String txtFilePath = args[0];
+
+        File file = new File(txtFilePath);
+        if(file.length()==0)
```

Comment:
What does file.length() do? Does it validates the file path? The error message inside if statement says "enter valid path". Please check.

Reply:
changed the print statement to  "the file does not contains any data".